css/style.css

```diff
@@ -111,6 +112,12 @@ video {
 .pointer-list__oval--black{
   background-color: #333333;
 }
+
+.pointer-section {
+  display: grid;
+  align-items: center;
+  justify-items: center;
+}
 @media screen and (max-width: 960px){
   .pointer-list__oval{
```

Comment:
Add a space between the class name and {

Reply:
in the next commit i will do that.